### PR TITLE
Bluetooth: controller: split: Refactor if_defines to is_enabled in lll.c

### PR DESCRIPTION
**Pull request by alexsven**
This commit refactors if defines to use the macro is_enabled,
for readability

Signed-off-by: Alexander Svensen <alsv@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c [hunk 1]
```diff
@@ -408,12 +410,12 @@ static int prepare(lll_is_abort_cb_t is_abort_cb, lll_abort_cb_t abort_cb,
 #endif /* CONFIG_BT_CTLR_LOW_LAT */
 		int ret;
 
-#if defined(CONFIG_BT_CTLR_LOW_LAT)
-		/* early abort */
-		if (event.curr.param) {
-			event.curr.abort_cb(NULL, event.curr.param);
+		if (IS_ENABLED(CONFIG_BT_CTLR_LOW_LAT)) {
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c]: Combine in to single if-clause

**Review by cvinayak — CHANGES_REQUESTED** [review]

**Review by joerchan — APPROVED** [review]

File: subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c [hunk 2]
```diff
@@ -221,16 +221,16 @@ void lll_disable(void *param)
 int lll_prepare_done(void *param)
 {
 #if defined(CONFIG_BT_CTLR_LOW_LAT) && \
```
Comment (thoh-ot) [inline subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c]: Use  IS_ENABLED here also?

Comment (alexsven) [reply subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c]: ticker_op_job_disable is declared in a #if define (line:55), so in that case only CONFIG_BT_CTLR_LOW_LAT would be in an IS_ENABLED, and LLL_PRIO and ULL_LOW_PRIO would have to be an #if define. Considered it better to not mix the two.

Comment (thoh-ot) [reply subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c]: OK

**Review by thoh-ot — APPROVED** [review]

**Comment by aescolar** [comment]
@carlescufi Please review and/or merge at will (this affecting only the Nordic LLL)

**Review by carlescufi — APPROVED** [review]

**Review by cvinayak — APPROVED** [review]

**Pull request merged**